src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+// Actions
+const STATUS = '/src/redux/categories/STATUS';
+
```

Comment:
- [ ] Good job so far Please make sure that your actions follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), according to that you have action types in the form` my-app/reducer/ACTION_TYPE` You have something like `const STATUS = 'Nyame-Bookstore/categories/STATUS'`. Please fix it :+1: